v2h_templates.py

```diff
@@ -122,6 +122,7 @@
 <main>
 <h1>Subway Validation Results</h1>
 <p>Total good metro networks: {good_cities} of {total_cities}.</p>
+<p>Please follow <a href="https://wiki.openstreetmap.org/wiki/Metro_Mapping">metro mapping instructions</a>.</p>
```

Comment:
A OSM mapper getting sent a link to the Validator should be able to understand from the front page what this page is about. Right now the first section with three sections feels a bit stitched together. Maybe we can combine the first two paragraphs wth something like
`<p>{good_cities} of {total_cities} networks validated without errors. To make a network validate successfully please follow the <a href="https://wiki.openstreetmap.org/wiki/Metro_Mapping">metro mapping instructions</a>. After editing the OSM data check back to the updated validation results on the next day to ensure your changes improved the quality and did not introduce new issues.</p>`